Python/symtable.c

```diff
@@ -2001,6 +2055,20 @@ symtable_visit_dictcomp(struct symtable *st, expr_ty e)
                                          e->v.DictComp.value);
 }
 
+static int
+symtable_check_annotation(struct symtable *st, const char *name, expr_ty e)
```

Comment:
This name is quite non descriptive, let's call this something like `symtable_raise_if_annotation_block` or something like that.